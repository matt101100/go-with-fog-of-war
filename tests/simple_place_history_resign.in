history
place A1
place B2
history
place C2
history
place S19
place S19
place S18
place S11
place S20
history
place Z1
history
place A2
history
resign
