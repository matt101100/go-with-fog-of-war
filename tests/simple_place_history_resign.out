
A1B2
A1B2C2
Occupied coordinate
Invalid coordinate
A1B2C2S19S18S11
Invalid coordinate
A1B2C2S19S18S11
A1B2C2S19S18S11A2
Black wins!
A1B2C2S19S18S11A2
Thank you for playing!
